components/clarinet-deployments/src/types.rs

```diff
@@ -586,6 +765,7 @@ impl EmulatedContractCallSpecification {
 #[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
 pub struct EmulatedContractPublishSpecification {
```

Comment:
Note: verify in the same method as was done here: https://github.com/hirosystems/clarinet/pull/1116/files#r1297274266

Reply:
Verified. Everything compiled after derives exclusion.